main.go

```diff
@@ -84,13 +102,32 @@ type jiraClient struct {
 	client *jira.Client
 }
 
+type loggingRoundTripper struct {
+	r http.RoundTripper
+}
+
+func (f *loggingRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
+	// command, _ := http2curl.GetCurlCommand(r)
+	// fmt.Println(command)
```

Comment:
```suggestion
```

Reply:
I left it as I find it useful for debugging - happy to delte.

Reply:
There is otherwise no sue for the custom RoundTripper then - so I'll remove and squash